docs/pages/Configuration.md

```diff
@@ -750,6 +751,15 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- Remove bundle option [`usePredefineCalls`](#properties) 
+- New option “async“ for [builder.bundles.bundleDefinition.section](#properties)
```

Comment:
The async flag is not yet documented at the linked chapter. Should we do that in this PR or separately?

Reply:
Thanks!
I'll add it with this PR

Reply:
Done